Expensify/App
# Exclude concierge from iou

@trjExpensify 

### Details
Exclude Concierge from IOU participant selection and remove IOU menu items from chat with Concierge.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify.cash/issues/2698

### Tests
Tested manually and made sure that users cannot initiate IOU request with Concierge.

### QA Steps
```
- Open 1:1 and group chat with Concierge
- Press `+` icon next to chat composer
- Observe that there is no `Request Money` or `Split Bill` option on menu
```

```
- Open 1:1 and group chat without Concierge
- Press `+` icon next to chat composer
- Observe that there is a `Request Money` or `Split Bill` option on menu
```

```
- Press big green `+` icon on the sidebar
- Select `Request Money` or `Split Bill`
- Enter a random amount and press `Next`
- Search Concierge
- Observe that Concierge doesn't appear among results
```

### Tested On


### Screenshots
#### Desktop
<img width="400" alt="Screenshot 2021-05-07 at 17 13 19" src="https://user-images.githubusercontent.com/3853003/117480915-248bef00-af5a-11eb-8a59-044da042a883.png"> | <img width="400" alt="Screenshot 2021-05-07 at 17 13 50" src="https://user-images.githubusercontent.com/3853003/117480925-26ee4900-af5a-11eb-8a7e-a75882743bc5.png">
<img width="400" alt="Screenshot 2021-05-07 at 17 14 23" src="https://user-images.githubusercontent.com/3853003/117480927-2786df80-af5a-11eb-8d25-20fe009b5286.png"> | <img width="400" alt="Screenshot 2021-05-07 at 17 14 59" src="https://user-images.githubusercontent.com/3853003/117480928-28b80c80-af5a-11eb-8433-6b8d7d298461.png">
<img width="400" alt="Screenshot 2021-05-07 at 17 15 43" src="https://user-images.githubusercontent.com/3853003/117480932-2950a300-af5a-11eb-9a95-d81e88720cf5.png"> | <img width="400" alt="Screenshot 2021-05-07 at 17 16 20" src="https://user-images.githubusercontent.com/3853003/117480935-29e93980-af5a-11eb-8a3a-701b0c454d5d.png">
